apps/prairielearn/python/prairielearn.py

```diff
@@ -1058,7 +1058,7 @@ def string_to_integer(s: str, base: int = 10) -> int | None:
         return None
 
 
-def string_to_number(s, allow_complex=True):
+def string_to_number(s, *, allow_complex=True):
```

Comment:
Need to check `string_to_number` callsites outside of our codebase.

Reply:
Confirmed that there's only one use of this function in the wild and it's already using a named argument.